tests/s3_bucket/test_s3_pages_content.py

```diff
@@ -75,21 +77,31 @@ def test_page_content(bucket_books_tree, s3_books_title_version_dict, code_tag,
     for elem in range(len(bucket_books_tree)):
 
         book_id.clear()
-        book_id.append(bucket_books_tree[elem]["id"])
+        uuids_vers = bucket_books_tree[elem]["id"]
+        book_id.append(uuids_vers[0 : uuids_vers.index("@")])
 
         book_title.clear()
         book_title.append(bucket_books_tree[elem]["title"])
 
         title_results = list(s3_books_title_version_dict.keys())
 
+        try:
+
+            # compares book uuids in the queue-state list against actual books in aws s3 bucket
+            assert [j for j in book_id if j in s3_books_uuids]
```

Comment:
Just to confirm my understanding: The goal of this assertion is to fail if a book UUID is in the bucket but not in the queue state list, correct? If so, I think the problem is `bucket_books_tree` itself is generated based upon `s3_queue_state_bucket_books`, so is it even possible for this to happen in terms of the code? (e.g. I think to really check for this you'd have to do an `s3 ls` equivalent on the bucket)